desktop/app/kbfs-helper.js

```diff
@@ -1,10 +1,32 @@
 // @flow
 
 import {ipcMain, shell} from 'electron'
+import fs from 'fs'
+import {pathToURL} from './paths'
 
 export default function () {
   ipcMain.on('openInKBFS', (e, path) => {
-    // We do the `shell.openItem` here to not block the ui thread
-    shell.openItem(path)
+    fs.stat(path, (err, stats) => {
+      if (err) {
+        console.warn('Error opening %s:', path, err)
+        return
+      }
+      if (stats.isFile()) {
+        shell.showItemInFolder(path)
+      } else if (stats.isDirectory()) {
+        // Paths if directories might be symlinks.
+        // For example /keybase/private/gabrielh,chris gets redirected to
+        // /keybase/private/chris,gabrielh.
+        fs.realpath(path, (err, resolvedPath) => {
+          if (err) {
+            console.warn('No realpath for %s:', path, err)
```

Comment:
`openItem` doesn't focus the Finder, so the UX feels a little weird. Also `openItem` uses old AppleScript APIs to workaround some 10.8 bugs, so that's not great especially since we only need 10.10 and above.

Also this change would allow us to open files (that aren't dirs) as selected in the finder, which is a little safer.